crates/agglayer-settlement-service/src/settlement_task.rs

```diff
@@ -0,0 +1,12 @@
+/// Internal configuration for settlement tasks.
+/// This is not exposed to users and is configured programmatically by the
+/// service.
+pub struct SettlementTaskConfig {}
```

Comment:
Does it needs to be `pub` if it's not exposed ?

Reply:
Maybe not. We will iterate on this one. We may need it public for the integration tests.